localized/ja/UserGuide-chapter1.adoc

```diff
@@ -0,0 +1,149 @@
+= Chapter 1. OptaPlanner イントロダクション
+:awestruct-description: Chapter 1. OptaPlanner イントロダクション
+:awestruct-layout: localizedBase
+:awestruct-lang: jp
+:awestruct-priority: 1.0
+:showtitle:
+xxx
+
+////
+
+Failed, to be deleted soon >>>>>
```

Comment:
Dear Geoffrey, 

Thank you for your managing the Japanese localized files. 
Also, thank you for the information on how to change the text placed in images based on the SVG format. I will try that. 

I believe the quality of the present translation would well be accepted, hoping more sections are collaboratively added to achieve our first goal. 

To tell the truth, the PR I sent was a result of my careless operation failure I made as a beginner while I was browsing around the related GitHub website. Actually, I had to keep the version within our Japanese branch, which will continuously be maintained by Kobayashi-san. The final version will be prepared for the upstream PR by him after collecting and unifying all the files created by local contributors. In that process, the needless comments you found in the temporary file will also be deleted. 

—

This is a super tiny correction in the German index.html I found in the localization folder today: 
In sentence 4: Produkte un Dienste —> Produkte und Dienste

Best regards, 
Takugo Ishii





> 2015/09/08 16:20、Geoffrey De Smet <notifications@github.com> のメール：
> 
> In localized/ja/UserGuide-chapter1.adoc:
> 
> > @@ -0,0 +1,149 @@
> > += Chapter 1. OptaPlanner イントロダクション
> > +:awestruct-description: Chapter 1. OptaPlanner イントロダクション
> > +:awestruct-layout: localizedBase
> > +:awestruct-lang: jp
> > +:awestruct-priority: 1.0
> > +:showtitle:
> > +xxx
> > +
> > +////
> > +
> > +Failed, to be deleted soon >>>>>
> 
> I presume a second PR is coming to delete these comments? I'll merge the current PR already.
> 
> Thank you for contributing this translation! Great work.
> If you want to be recognized as a contributor, please submit a PR to add yourself in this file:
> https://github.com/droolsjbpm/optaplanner-website/blob/master/_config/users.yml
> 
> —
> Reply to this email directly or view it on GitHub.
>